source/_docs/change-management.md

```diff
@@ -180,3 +180,11 @@ In some Dashboards, you may notice the "User in Charge" label applied to a user.
         <td><center>X</center></td>
     </tr>
 </table>
+
+## Frequently Asked Questions (FAQs)
+
+#### Which role should I assign a user to give them the lowest level of access?
```

Comment:
Yes, this is correct at site level. At org level, Unprivileged is lowest level of access.